coordinator/src/services/handlers/sandbox-handler.ts

```diff
@@ -45,7 +45,7 @@ export class SandboxRequestHandler implements RequestHandler {
     } else {
       return Promise.resolve({
         statusCode: 400,
-        body: notSupporedOperationOutcome
+        body: notSupportedOperationOutcome
```

Comment:
why does git think there have been any changes in this file?

Reply:
because i've fixed a typo in this const name haha

Reply:
ohhh lol yea i see it